app/controllers/users_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+class UsersController < ApplicationController
+  def index
+    @users = User.page(params[:page]).per(20)
```

Comment:
Move them in this pull request or in future?
And if now, make new controller for /account actions? Or use just "scope '/account' do..."?